File: TraningAPI/View/LicenceListView/LicenceTableViewCell.m
//
//  LicenceTableViewCell.m
//  TraningAPI
//
//  Created by 高橋慧 on 2021/06/23.
//  Copyright © 2021 k-takahashi. All rights reserved.
//

#import "LicenceTableViewCell.h"

@implementation LicenceTableViewCell

+ (NSString *)cellIdentifer {
    return @"LicenceTableViewCell";
}
- (void)awakeFromNib {
    [super awakeFromNib];
}
- (void)prepareForReuse {
    [super prepareForReuse];
    self.licenceNameLabel.text = nil;
}
- (void)licenceNameConfigure:(NSString *)text {
    self.licenceNameLabel.text = text;
}
@end

File: TraningAPI/View/LicenceListView/LicenceTableViewCell.swift
//
//  LicenceTableViewCell.swift
//  TraningAPI
//
//  Created by 高橋慧 on 2021/04/30.
//  Copyright © 2021 k-takahashi. All rights reserved.
//

import UIKit

class LicenceTableViewCell: UITableViewCell {
    static let className = "LicenceTableViewCell"
    @IBOutlet weak var licenceNameLabel: UILabel!
    
    override func awakeFromNib() {
        super.awakeFromNib()
        // Initialization code
    }

    override func setSelected(_ selected: Bool, animated: Bool) {
        super.setSelected(selected, animated: animated)

        // Configure the view for the selected state
    }
    override func prepareForReuse() {
        super.prepareForReuse()
        licenceNameLabel.text = nil
    }
    
    internal func licenceNameConfigure(text: String){
        licenceNameLabel.text = text
    }
    
}
